ivy/functional/frontends/jax/lax/linalg.py

```diff
@@ -10,3 +10,17 @@ def symmetrize(x):
         x = symmetrize(x)
 
     return ivy.cholesky(x)
+
+
+
+
+
+def matrix_power(m: Union[ivy.Array, ivy.NativeArray],
```

Comment:
There should not be any type hints in the frontend functions. Delete the type-hint related imports too.